test/test_jit.py

```diff
@@ -7211,6 +7232,10 @@ def script_fn(*args, **kwargs):
             raise 'Unsupported function type'
 
         script = script_template.format(', '.join(formals), call)
+
+        for module in modules:
+            locals()[module] = importlib.import_module(module)
```

Comment:
There has to be some import here since this is auto-generating tests from `test_autograd.py`, or unconditionally import `math` even for tests that don't need it